demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -883,6 +884,22 @@ def is_there_a_runnable(self) -> bool:
             return False
         return True
 
+    def is_valid_parameters_display_name(self) -> bool:
+        configuration = self.current_file.get('configuration', {})
+        parameters_display_name = [param.get('display') for param in configuration if param.get('display')]
+
+        invalid_display_names = []
+        for parameter in parameters_display_name:
+            invalid_display_names.append(parameter) if parameter and not parameter[0].isupper() or '_' in parameter \
+                else None
+
+        if invalid_display_names:
+            error, code = Errors.invalid_integration_parameters_display_name(invalid_display_names)
```

Comment:
how will the error look?
conside join with `, ` the invalid_display_names

Reply:
It looks like a list of problematic words -
The integration display names: `[username, Client_id]` are invalid, Integration parameters display name should be capitalized and spaced using whitespace and not underscores."